runtime/lib/app-runtime.js

```diff
@@ -600,11 +600,15 @@ AppRuntime.prototype.setMicMute = function setMicMute (mute, options) {
   }
 
   return future
-    .then(() => this.component.light.play(
-      '@yoda',
-      'system://setMuted.js',
-      { muted: muted },
-      { shouldResume: muted }))
+    .then(() => {
+      var noTts = typeof this.component.lifetime.activeSlots.scene === 'string' &&
```

Comment:
Simple as `!!this.component.lifetime.activeSlots.scene` might be ok.

Reply:
OK.